Gemfile

```diff
@@ -1,3 +1,4 @@
+ruby RUBY_VERSION
```

Comment:
thanks, something was complaining about that

Reply:
oh whoops, this snuck in there, I didn't mean to commit this